e2e/tests/cohorts/cohort-create.spec.ts

```diff
@@ -189,42 +184,8 @@ describe('Create Cohorts from Program Data criteria', () => {
     await new CohortActionsPage(page).deleteCohort();
   });
 
-  // Include all visit types in a single Include Group.
-  test('Create cohort from visits', async () => {
```

Comment:
Since `cohort-edit.spec` test needs to work with a new Cohort. I'm moving this `test('Create cohort from visits')` to `cohort-edit.spec`. This makes this test smaller, faster to run. No loss of test coverage.